frontend/public/locales/en/gcweb.json

```
@@ -21,6 +21,7 @@
     "menu-title": "Account",
     "menu-dashboard.text": "My Dashboard",
     "menu-dashboard.href": "{{baseUri}}/en/my-dashboard",
+    "menu-msca-home.href": "{{baseUri}}/sc/msca-mdsc/portal-portail/pro/home-accueil?Lang={{lang}}",
```

Can we not put the `lang` right into the URL?

```suggestion
    "menu-msca-home.href": "{{baseUri}}/sc/msca-mdsc/portal-portail/pro/home-accueil?Lang=en",
```

Same comment for `/fr/gcweb.json`.

---

fixed